use-cases/client-server-msgs-perf/svmsg_file.h

```diff
@@ -24,29 +24,52 @@
 
 #define SERVER_KEY 0x1aaaaaa1           /* Key for server's message queue */
 
-struct requestMsg {                     /* Requests (client to server) */
-    long mtype;                         /* Unused */
-    int  clientId;                      /* ID of client's message queue */
-    char pathname[PATH_MAX];            /* File to be returned */
-};
-
-/* REQ_MSG_SIZE computes size of 'mtext' part of 'requestMsg' structure.
-   We use offsetof() to handle the possibility that there are padding
-   bytes between the 'clientId' and 'pathname' fields.
-*/
+#define MAX_CLIENTS 64
+
+/**
+ * Types for request / response messages sent from server to client
+ */
+typedef enum {
+      REQ_MT_UNKNOWN    = 0
+    , REQ_MT_INIT       = 1     // New client initialization
+    , REQ_MT_INCR               // Increment client's counter
```

Comment:
Currently client only requests this op-type; i.e. increment.

Rest are added for possible future usages, but really serve no much purpose.